databroker/core.py

```diff
@@ -61,14 +61,13 @@ def from_run_start(cls, db, run_start):
         run_start_uid = run_start['uid']
 
         try:
-            run_stop = doct.ref_doc_to_uid(mds.stop_by_start(run_start_uid),
-                                           'run_start')
+            run_stop = mds.stop_by_start(run_start_uid)
         except mds.NoRunStop:
             run_stop = None
 
-        d = {'start': run_start}
+        d = {'start': db.prepare_hook('start', run_start)}
         if run_stop is not None:
-            d['stop'] = run_stop
+            d['stop'] = db.prepare_hook('stop', run_stop)
```

Comment:
Reasonable. We used to say that if the RunStop was missing we just wouldn't put the key in. Now it looks like we make it an empty dict by default? Fair enough....